scripts/o3de/ExportScripts/export_source_built_project.py

```diff
@@ -87,28 +87,35 @@ def export_standalone_project(ctx: exp.O3DEScriptExportContext,
         logger.setLevel(logging.ERROR)
 
     # Calculate the tools and game build paths
-    default_base_build_path = ctx.engine_path / 'build' if engine_centric else ctx.project_path / 'build'
+    default_base_path = ctx.engine_path if engine_centric else ctx.project_path
     
     if is_installer_sdk:
-            # Check if we have any monolithic entries if building monolithic
-            monolithic_artifacts = glob.glob(f'cmake/Platform/{exp.get_platform_installer_folder_name()}/Monolithic/ConfigurationTypes_*.cmake',
-                      root_dir=ctx.engine_path)
-            has_monolithic_artifacts = len(monolithic_artifacts) > 0
-
-            if monolithic_build and not has_monolithic_artifacts:
-                logger.error("Trying to create monolithic build, but no monolithic artifacts are detected in the engine installation! Please re-run the script with '-nomonolithic' and '-config profile'.")
-                return 1
-            tools_build_path = ctx.engine_path / 'bin' / exp.get_platform_installer_folder_name(selected_platform) / tool_config / 'Default'
+        # Check if we have any monolithic entries if building monolithic
+        if monolithic_build and not exp.has_monolithic_artifacts(ctx):
+            logger.error("Trying to create monolithic build, but no monolithic artifacts are detected in the engine installation! Please re-run the script with '-nomonolithic' and '-config profile'.")
+            return
+        tools_build_path = ctx.engine_path / 'bin' / exp.get_platform_installer_folder_name(selected_platform) / tool_config / 'Default'
     
+    # Note, when resolving the relative paths used for the export process, there is a debate about 
+    # whether to use the current working directory, or to forcefully set the base path to an O3DE directory.
+    # In practice, there was noticeable surprise whenever the relative folder wasn't found in an O3DE directory,
+    # and sometimes commands are carelessly run from sensitive directories (such as C:\\Windows\\System32)
```

Comment:
Do you have data of users having issues with specifying relative paths?

Reply:
atm no recorded data. This was based on experiences with @AMZN-Gene and @AMZN-alexpete when they were using the export system to build out sample projects. I adjusted this script based on what I noticed were common trouble points they experienced.

Reply:
Correct, I export from whatever folder my terminal brings up by default and pass in absolute paths when parameters require a path.